### ISSUE TITLE
As a user, I would should be informed when I no longer need to keep tip confirmation window open

### ISSUE THREAD

**Issue by jakerockland** [gitcoinco/web#928]
### User Story

[comment]: # (As a <user type>, I want to <task> so that <goal>.)
As a user, I would should be informed when I no longer need to keep tip confirmation window open.

### Why Is this Needed?

[comment]: # (Describe the problem and why this task is needed. Provide description of the current state, what you would like to happen, and what actually happen)
*Summary*: Currently after submitting public address to claim a tip, without a knowledge of Web3, it is unclear that I can close the browser window while the transaction confirms.

### Description

[comment]: # (Feature or Bug? i.e Type: Bug)
*Type*: Bug

### Current Behavior
[comment]: # (Describe what actually happened.)
<img width="1440" alt="screenshot 2018-04-20 09 39 30" src="https://user-images.githubusercontent.com/8602661/39060611-115b2426-447f-11e8-8419-41ddfa3c561c.png">

### Expected Behavior
[comment]: # (Describe what you expected to happen.)
The above screen should inform me as a user that I can close the tab.

### Reproduction
[comment]: # (Describe how we can replicate the bug step by step.)
Claim a tip.

### Solution
[comment]: # (Provide a summary of the solution and a task list on what needs to be fixed.)
*Summary*: Updated copy of the above screen.

**Comment by owocki** [comment]
thanks for the suggestion.  woudl adding some paragraph text that says 'you can close your browser window' to the page help?

**Comment by jakerockland** [comment]
@owocki yeah I think that's just noting that in a tiny paragraph under "See your transaction on the blockchain here" would be ideal 👌
